frontend/common/.storybook/decorators/ThemeDecorator.tsx

```diff
@@ -1,12 +1,78 @@
 import React from "react";
-import type { StoryFn } from "@storybook/react";
+import type { StoryContext, StoryFn } from "@storybook/react";
+import isChromatic from "chromatic/isChromatic";
 import ThemeProvider from "../../src/components/Theme";
 
-// Applies the ThemeContext to a story
-export default function ThemeDecorator(Story: StoryFn) {
-  return (
-    <ThemeProvider>
+export const theme = {
+  name: "Theme",
+  description: "Global theme for components",
+  defaultValue: "light",
+  toolbar: {
+    icon: "circlehollow",
+    // Array of plain string values or MenuItem shape (see below)
+    items: [
+      {
+        value: "pref",
+        right: "🖥",
+        title: "Preference",
+      },
+      {
+        value: "light",
+        right: "☀️",
+        title: "Light",
+      },
+      {
+        value: "dark",
+        right: "🌙",
+        title: "Dark",
+      },
+    ],
+    dynamicTitle: true,
+  },
+};
+
+interface ThemeBlockProps {
```

Comment:
Unused interface?

Reply:
Removed with [remove unused interface](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4229/commits/aa9b1c29e96ab6d95fdb41548888433c6ef77d8f)

And, you can probably guess what was causing the home page forced to dark with the name of the fix: [revert theme decorator debugging](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4229/commits/0a03049131f69292bc4396974b0d08e5e0209b48)